jovian/jovian_nb_ext/main.js

```diff
@@ -707,6 +861,48 @@ define([
         .modal("show");
     };
 
+    // disables the extension
+    function removeExtension() {
+      Jupyter.notebook.save_checkpoint();
+      remove_ext =
+        "import os\n" +
+        "os.system('jupyter nbextension disable jovian_nb_ext/main --sys-prefix')\n";
+      console.log(remove_ext);
+      alert(
+        "You have disabled the Jovian Extension. Run -- !jovian enable -- in the notebook to renable Jovian"
```

Comment:
Can you remove `--`.

Also the correct command is `!jovian enable-extension` can you change that too.

Can you add `extension` to the end of the sentence.